sw/device/lib/dif/dif_rstmgr_unittest.cc

```diff
@@ -35,8 +35,11 @@ TEST_F(ResetTest, NullArgs) { EXPECT_DIF_BADARG(dif_rstmgr_reset(nullptr)); }
 TEST_F(ResetTest, Success) {
   EXPECT_WRITE32(RSTMGR_RESET_INFO_REG_OFFSET,
                  std::numeric_limits<uint32_t>::max());
-  EXPECT_WRITE32(RSTMGR_SW_RST_CTRL_N_REG_OFFSET,
-                 std::numeric_limits<uint32_t>::max());
+
+  for (uint32_t i = 0; i < RSTMGR_PARAM_NUM_SW_RESETS; i++) {
```

Comment:
Prefer preincrement, i.e. `++i`.

Reply:
dumb question, can you explain the preference here a bit..?

Reply:
Pre-increment returns the value of i after incrementing it, whereas post-increment needs to create a temporary variable to return the value as it was before incrementing.